dll/cpl/input/lang/pt-BR.rc

```diff
@@ -22,7 +22,7 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Configurações de sistema", -1, 7, 7, 240, 70
     AUTOCHECKBOX "&Desabilitar serviços de texto avançado", IDC_TURNOFFTEXTSVCS_CB, 14, 20, 220, 12
-    LTEXT "Marque esta caixa de seleção para deslitar os serviços de texto avançado em todos os programas. Isto não é recomendado para usuários do Leste Asiático por que isto irá fechar a barra de idiomas.", -1, 14, 40, 230, 35
+    LTEXT "Marque esta caixa de seleção para desabilitar os serviços de texto avançado em todos os programas. Isto não é recomendado para usuários do Leste Asiático por que isto irá fechar a barra de idiomas.", -1, 14, 40, 230, 35
```

Comment:
A suggested improvement, from discussion with a native Brazilian portuguese person:
> (Mysoft) i would just change
> (Mysoft) "por que isto irá fechar a barra de idiomas."
> (Mysoft) to
> (Mysoft) "por que irá fechar a barra de idiomas."
> (Mysoft) the "isto" there is redundant
> (Mysoft) like
> (Mysoft) "that is not recommended because that will close the language bar"
> (Mysoft) on english the second "that" is normally used
> (Mysoft) but in portuguese it's redundant

Reply:
yes, without the second "isto" is better

Reply:
Thanks. Let me know when you update your commit.

Reply:
Update my commit?

Reply:
Yeah, removing the second "isto" and committing again (the pull request will automatically update).